recipes/app1.rb

```diff
@@ -137,3 +137,10 @@
     '8082 -c config/unicorn.rb -E deployment -D'
   end
 end
+
+# Logrotate
+logrotate_app 'OpenID' do
+  path '/home/openid-production/shared/log/*'
+  frequency 'daily'
+  rotate 30
+end
```

Comment:
What command is going to be run to reload the daemon? Otherwise you're going to have open file handles and issues with that.